packages/wrk/Makefile.patch

```diff
@@ -0,0 +1,29 @@
+--- a/Makefile
++++ b/Makefile
+@@ -32,7 +32,7 @@
+ LDFLAGS += -L$(ODIR)/lib
+ 
+ ifneq ($(WITH_LUAJIT),)
+-	CFLAGS  += -I$(WITH_LUAJIT)/include
++	CFLAGS  += -I$(WITH_LUAJIT)/include/luajit-2.1
+ 	LDFLAGS += -L$(WITH_LUAJIT)/lib
+ else
+ 	CFLAGS  += -I$(ODIR)/include/luajit-2.1
+@@ -62,7 +62,7 @@
+ 
+ $(ODIR)/bytecode.o: src/wrk.lua
+ 	@echo LUAJIT $<
+-	@$(SHELL) -c 'PATH=obj/bin:$(PATH) luajit -b $(CURDIR)/$< $(CURDIR)/$@'
++	@$(SHELL) -c 'PATH=obj/bin:$(PATH):$(TERMUX_PREFIX)/bin luajit -b $(CURDIR)/$< $(CURDIR)/$@'
```

Comment:
Should be `PREFIX`, not `TERMUX_PREFIX`.

Reply:
No, wait, this runs on host, so `@TERMUX_PREFIX@`.

Reply:
So how x86_64 host will execute binary of different arch ?